evaluator/evaluator.ml

```diff
@@ -53,6 +53,12 @@ end
 
 let setup () = JsooTop.initialize ()
 
+let insertModule name content = 
+  let name = Js.to_string name in
+  let content = Js.to_string content in
+  let _ = File__System.createOrUpdateFile name content in
+  Execute.mod_use_file ("/static" ^ name)
```

Comment:
How would you know if a module is successfully inserted from Bucklescript side?

Reply:
Alright. You could potentially return the result as Javascript readable form ? I have a converter in execute function to convert value to Js.t . I'll pull this branch locally and see if we can catch the errors or something